### PR TITLE
initial alpine musl support

### PR DESCRIPTION
**Pull request by gdams**

### CODE REVIEW

File: dockerhub_doc_config_update.sh [hunk 1]
```diff
@@ -35,7 +35,7 @@ fi
 wget -q -O official-eclipse-temurin https://raw.githubusercontent.com/docker-library/official-images/master/library/eclipse-temurin
 
 # TODO When we have Apline musl builds we should add alpine to the follow list
```
Comment (tianon) [inline dockerhub_doc_config_update.sh:37]: :eyes:

File: dockerhub_doc_config_update.sh [hunk 2]
```diff
@@ -191,7 +191,7 @@ rm -f ${official_docker_image_file}
 print_official_header
 
 # TODO once we have musl based alpine images we can remove alpine
```
Comment (tianon) [inline dockerhub_doc_config_update.sh:193]: :eyes:

File: dockerhub_doc_config_update.sh (hunk 1 above)
Comment (karianna) [reply dockerhub_doc_config_update.sh:37]: We're working on backporting to Java 11 and 8 - it'll be several weeks but we'll get there!

Comment (karianna) [reply dockerhub_doc_config_update.sh:37]: Oh, I see, comment is redundant at least with regards to this config.

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**